src/components/Tooltip/__stories__/Placement.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import { DecoratorFunction } from '@storybook/addons'
+import { ComponentProps } from 'react'
+import Tooltip from '..'
+
+export const Placement = (props: ComponentProps<typeof Tooltip>) =>
+  ['auto', 'top', 'bottom', 'left', 'right'].map(placement => (
+    <Tooltip
+      {...props}
+      key={placement}
+      placement={placement as ComponentProps<typeof Tooltip>['placement']}
```

Comment:
We should be able to avoid the cast by narrowing the array:

```suggestion
  (['auto', 'top', 'bottom', 'left', 'right'] as const).map(placement => (
    <Tooltip
      {...props}
      key={placement}
      placement={placement}
```

Or we could even type the array properly:
```suggestion
const PLACEMENTS: Array<ComponentProps<typeof Tooltip>['placement']> = ['auto', 'top', 'bottom', 'left', 'right']

  PLACEMENTS.map(placement => (
    <Tooltip
      {...props}
      key={placement}
      placement={placement}
```